doc/plotting/index.rst

```diff
@@ -0,0 +1,20 @@
+.. include:: ../tune_toc.rst
+
+
+.. _plotting:
+
+=========================
+Visualization
+=========================
+
+In this section, we detail the general tools to visualize
+neuroimaging volumes with nilearn.
+
+|
+
+.. include:: ../includes/big_toc_css.rst
+
+
+.. toctree::
+
+   plotting.rst
```

Comment:
I am not sure that this extra level of indirection is useful. Why don't you just include the contents of plotting.rst here (and thus remove the different tune_toc doc)?